CMP1903_A1_2324/Die.cs

```diff
@@ -1,23 +1,22 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
-
-        //Method
+        private int _dieValue; //creates a private variable for _dieValue, so that it couldn't be used by other c# files linked
+        private static Random rnd = new Random(); //creates a private and static random element, which is only going top be used in one area
+        private static Testing test = new Testing(); //A new object for Testing is created here, so that tests can be made as soon as possible
 
+        public int DieValue //A public variation of _dieValue is created here, so that it can be used elsewhere
```

Comment:
Good way to get a return as specified by the requirement, this will definitely be used later for the game.cs porgram